main.go

```diff
@@ -332,13 +336,75 @@ Note: may not work with some transports such as p2p-circuit (not applicable) and
 							}
 							return vole.Ping(c.Context, c.Bool("force-relay"), ai)
 						},
+					}, {
+						Name:      "http",
+						ArgsUsage: "<multiaddr>",
+						Flags: []cli.Flag{
+							&cli.StringFlag{
+								Name:        "socket-path",
+								Usage:       `Use the specified path for the unix socket instead of making a new one.`,
+								DefaultText: "",
+								Value:       "",
+							},
+						},
+						Usage: "Make http requests to the given multiaddr with a unix socket",
+						Description: `This command creates a unix socket that can be used with curl to make HTTP requests to the provided multiaddr.
```

Comment:
I might be missing something but given this is under the "libp2p" command I wonder if it's worth explicitly mentioning that this will enable setting up a standard HTTP proxy that uses multiaddr but otherwise doesn't use any libp2p specs at all (i.e. listening on an http multiaddr).

Reply:
It does HTTP over streams in the spec defined way?